esi/include.go

```diff
@@ -52,6 +53,12 @@ func (i *includeTag) Process(b []byte, req *http.Request) ([]byte, int) {
 		return nil, len(b)
 	}
 
+	if strings.HasPrefix(i.src, "//") {
```

Comment:
You should probably use [`URL.ResolveReference()`](https://pkg.go.dev/net/url#URL.ResolveReference) instead

Reply:
That doesn't work.

Reply:
Because in the caddy's context, req.URL.Host is empty and only req.Host is filled.

Reply:
I mean you can build the base URL instance by yourself, but relying on this method will be safer as it will support all edge cases